doc/docs/master/enterprise/auth/enterprise-server/setup.md

```diff
@@ -20,9 +20,58 @@ The setup of an Enterprise Server requires to:
 ## 1 - Deploy An Enterprise Server
 
 ### Single-cluster deployment
-Deploying a Pachyderm cluster with the embedded enterprise server does not require any specific action.
-Proceed as usual:
+Deploying and configuring a Pachyderm cluster with the embedded enterprise server can be done in one of two flavors.
+
+1. Provide all licensing and authentication configuration as a part of the Helm deployment.
+1. Install Pachyderm as usual with Helm, and use `pachctl` commands to set up licensing and authentication.
+
+#### To setup enterprise features as a part of the helm deployment
+Update your values.yaml with your enterprise license key, and auth configurations ([see example of values.yaml here](https://github.com/pachyderm/pachyderm/blob/master/etc/helm/examples/local-values.yaml)) or use our minimal example below.
+
+=== "values.yaml with activation of an enterprise license and authentication"
+
+	```yaml
+	pachd:
+		enterpriseLicenseKey: "<ENTEPRRISE-LICENSE-KEY>"
+		oauthClientId: "pachd"
+		oauthIssuer: "http://<PACHD-IP>:30658/"
+		oauthRedirectURI: "http://<PACHD-IP>:30657/authorization-code/callback"
+  		## if a secret is not provided, a secret will be autogenerated on install and stored in the k8s secret 'pachyderm-bootstrap-config.authConfig.clientSecret'
+  		oauthClientSecret: ""
+  		## if a secret is not provided, a secret will be autogenerated on install and stored in the k8s secret 'pachyderm-bootstrap-config.enterpriseSecret'
+  		enterpriseSecret: ""
+  		## if a token is not provided, a secret will be autogenerated on install and stored in the k8s secret 'pachyderm-bootstrap-config.rootToken'
+  		rootToken: ""
+  		## userAccessibleOauthIssuerHost is necessary in localhost settings or anytime the registered Issuer address isn't accessible outside the cluster
+  		# userAccessibleOauthIssuerHost: "localhost:30658"
+		## if `mockIDP` is set to true, `pachd.upstreamIDPs` will be ignored in favor of a testing placeholder IDP with username/password: admin/password
+  		mockIDP: true
+  		## to set up upstream IDPs, set pachd.mockIDP to false,
+  		## and populate the pachd.upstreamIDPs with an array of Dex Connector configurations.
+  		## See the example below or https://dexidp.io/docs/connectors/
+  		upstreamIDPs:
+  		  - id: idpConnector
+  		    jsonConfig: >-
+  		      {
+  		          "issuer": "<ISSUER>",
+  		          "clientID": "<CLIENT-ID>",
+  		          "clientSecret": "<CLIENT-SECRET>",
+  		          "redirectURI": "http://<PACHD-IP>:30658/callback",
+  		          "insecureEnableGroups": true,
+  		          "insecureSkipEmailVerified": true,
+  		          "insecureSkipIssuerCallbackDomainCheck": true,
+  		          "forwardedLoginParams": ["login_hint"]
+  		      }
+  		    name: idpConnector
+  		    type: oidc
+	```
```

Comment:
it may be out of the scope of this doc, but we could add the `console` configuration options here, so that the user can just copy this directly into their values file and deploy our whole stack

Reply:
Discussed this with Alon. The installation of the Console will be documented in a separate page